buildscripts/gpuci/build.sh

```diff
@@ -0,0 +1,64 @@
+##############################################
+# Numba GPU build and test script for CI      #
+##############################################
+set -e
+NUMARGS=$#
+ARGS=$*
+
+# Arg parsing function
+function hasArg {
+    (( ${NUMARGS} != 0 )) && (echo " ${ARGS} " | grep -q " $1 ")
+}
+
+# Set path and build parallel level
+export PATH=/opt/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}
+
+# Set home to the job's workspace
+export HOME="$WORKSPACE"
+
+# Switch to project root; also root of repo checkout
+cd "$WORKSPACE"
+
+# Determine CUDA release version
+export CUDA_REL=${CUDA_VERSION%.*}
+
+################################################################################
+# SETUP - Check environment
+################################################################################
+
+gpuci_logger "Check environment variables"
+env
+
+gpuci_logger "Check GPU usage"
+nvidia-smi
+
+gpuci_logger "Create testing env"
+. /opt/conda/etc/profile.d/conda.sh
+gpuci_mamba_retry create -n numba -y \
+                  "python=${PYTHON_VER}" \
+                  "cudatoolkit=${CUDA_TOOLKIT_VER}" \
+                  "llvmlite" \
```

Comment:
We need the development package of llvmlite for testing Numba master - I think this should do the right thing:

```suggestion
                  "numba/label/dev::llvmlite" \
```